devtools/conda-envs/test_min.yaml

```diff
@@ -1,17 +1,13 @@
 name: test
 channels:
   - conda-forge
-  - defaults
 dependencies:
   - python
   - pip
-  - numpy==1.20.0
+  - numpy==1.21.0
```

Comment:
I would suggest just removing 3.9 and bumping this up - but if folks are intent on keeping 3.9, then at least increasing to 1.21.0 is a need (since osx-arm64 will require 1.21+ for py39).

The main reason we're bumping here is because newer scipy will _work_ with older numpy, but features are mostly turned off (like pkdtree).

Reply:
I *think* 3.9 is included here because it’s the minimally supported version and that’s what this test is supposed to check.

Reply:
Sorry what I meant here was "maybe the minimum should be Python 3.10 not 3.9"

Reply:
If you want to bump the minimally supported Python version of membrane-curvature to 3.10 then we can do that  (even though its dependencies still support 3.9). If we bump then we should do a new release with advertised 3.10 - 3.12 (not sure if 3.12 is in pyproject.toml).

Reply:
It is in there — so it would just be removing 3.9 along spec0 Blabla.